rllib/tests/test_rllib_train_and_evaluate.py

```diff
@@ -193,6 +203,17 @@ def policy_fn(agent_id, episode, **kwargs):
             )
         )
 
+        # This is only supported without RLModule API. See AlgorithmConfig for
+        # more info.
+        if not config._enable_rl_module_api:
+            config.evaluation(
+                evaluation_config=AlgorithmConfig.overrides(explore=False)
```

Comment:
if explore=True for both it should not error, right? explore changes the behavior between picking forward_exploration vs. forward_sampling during sampling, so I don't think it should set differently.